helm-chart/templates/nextcloud-aio-onlyoffice-deployment.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if eq .Values.ONLYOFFICE_ENABLED "yes" }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.cmd: kompose convert -c -f latest.yml
+    kompose.version: 1.27.0 (b0ed6a2c9)
+  labels:
+    io.kompose.service: nextcloud-aio-onlyoffice
+  name: nextcloud-aio-onlyoffice
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: nextcloud-aio-onlyoffice
+  strategy:
+    type: Recreate
```

Comment:
Is this a good strategy?

Reply:
Yes, this is a reasonable default (especially for non-distributed applications). 

However, given that there is only one replica, it doesn't make much of a difference.